.env

```
@@ -7,3 +7,7 @@ DELETE_SINGLE_NOTIFICATION="notifications"
 MARK_NOTIFICATION_AS_READ_URL="notifications/mark-read"
 
 NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
+
+
```

dotenv-linter (3.3.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)